### PR TITLE
pinebookpro-kernel: update to 5.10.1

### PR DESCRIPTION
**Pull request by CameronNemo**
Related: #27346

[ci skip]

### CODE REVIEW

File: srcpkgs/pinebookpro-kernel/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'pinebookpro-kernel'
 pkgname=pinebookpro-kernel
-version=5.9.12
+version=5.10.1
```
Comment (ericonr) [inline srcpkgs/pinebookpro-kernel/template:3]: Why not .2?

Comment (CameronNemo) [reply srcpkgs/pinebookpro-kernel/template:3]: When I started this, it was at .1... I should have checked this morning :/

**Comment by ericonr** [comment]
Does this fix the IWD issues? I believe the IWD/ell testsuites can help here.

**Comment by CameronNemo** [comment]
@ericonr does not fix as far as I know. Looks like there is more support that iwd needs. See the issue filed for the errors I was seeing.

**Pull request merged**